thesis-gui/src/components/my-quizzes/MyQuizzes.tsx

```diff
@@ -29,22 +29,40 @@ const QuizzesListItem: React.FC<{
     questionsSetKey,
     reloadQuizzes
 }) => {
+    const COPY_KEY_TOAST_MESSAGE = `"${name}" quiz key was copied to clipboard!`;
+    const REFRESH_KEY_TOAST_MESSAGE = `"${name}" quiz key was refreshed and copied to clipboard!`;
+
     const { tryRefreshTokens } = useUserContext();
     const history = useHistory();
 
     const [present] = useIonToast();
 
-    const presentToast = () => {
+    const presentToast = (toastMessage: string) => {
         present({
-            message: `"${name}" quiz key was copied to clipboard!`,
+            message: toastMessage,
             duration: 1500,
-            position: 'bottom'
+            position: 'top'
         });
     };
 
-    const copyKeyToClipboard = () => {
-        navigator.clipboard.writeText(questionsSetKey);
-        presentToast();
+    const copyKeyToClipboard = (key: string, toastMessage: string) => {
```

Comment:
I started to wonder if the key should be displayed somewhere. Currently, the only way to see the key is to copy it and paste somewhere. What if the teacher wants to see the key in order to write it down on the board or something, so the students can see it? But it's not something that has to be done in this PR.

Reply:
I was also wandering about that but didn't find any good solution for that